packages/@lightningjs/ui-components/src/components/MetadataCardContent/MetadataCardContent.js

```diff
@@ -40,6 +40,10 @@ export default class MetadataCardContent extends MetadataBase {
         },
         Description: {
           type: TextBox
+        },
+        DescriptionDetails: {
+          mountY: -0.5,
```

Comment:
Is there a reason the `mountY` is negative instead of adding a spacing value? Is it always supposed to be spaced down half the height of the text?

Reply:
Yeah, I didn't see any other examples of a negative `mountY`.  Zero or undefined looked crowded but maybe that's better handled someplace else?

![image](https://github.com/rdkcentral/Lightning-UI-Components/assets/2964509/d9ab4b1f-d40f-488b-8a16-e74fcbae7309)

Reply:
Yes, I think it'd be better to use `_updateDescriptionDetails` (or a new method) to calculate what the position should be based on the height of the element(s) above it and add a `paddingY` style value.

Reply:
Looks great in general @AaronEdwardBegley. However, to @ImCoolNowRight 's point I think we might need to add more space another way. Can you explore paddingY or perhaps some line-height calculations based on the theme's  
fontSize?


<br class="Apple-interchange-newline">

Reply:
Cool, I added a `paddingY` style to descriptionDetails and that feels much better. @ImCoolNowRight & @joshhowenstine take a look and let me know what you think!